### ISSUE TITLE
Fail to load content with trailing spaces

### ISSUE THREAD

**Issue by adrianmo** [artonge/go-csv-tag#17]
The library fails to parse CSV files that contain lines with trailing spaces. I came across this issue when trying to read a GTFS file using the go-gtfs library.

E.g.:

```
header1, header2, header3
line1, 1, 1.2                                                                                                                                                                                                                                                                                                                         
line2, 2, 2.3                                                                                                                                                                                                                                                                                                                         
line3, 3, 3.4           
```

Error produced:

```
error mapping csv from path csv_files/trailingSpaces.csv:
        ==> error mapping the content to the destination
        ==> line: [line1 1 1.2                ] to slice: {line1 1 0}:
        ==> error parsing float '1.2                ':
        ==> strconv.ParseFloat: parsing "1.2                ": invalid syntax
```

**Comment by artonge** [comment]
Hi, thanks for the reporting !

Should be fixed with your #18. It is deployed in version go-csv-tag@v2.0.6 and go-gtfs@v1.0.1.

I will let you close this issue when you can check that the error is gone.

Thanks again !

**Comment by adrianmo** [comment]
The issue is now gone. Great job with these two libraries!
